tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -5,9 +5,9 @@ test_devs() {
   local devs=$TEST_DEVS
   local test_status=1
   local testname=`basename "$0"`
-  local vg_name="dss-test-foo"
+  local vg_name="css-test-foo"
```

Comment:
So you want me to just change file names in first pass and then internal references in second/third patch?